src/app/routes/utils/fetchPageData/index.js

```diff
@@ -14,28 +10,17 @@ import {
   NOT_FOUND,
   UPSTREAM_CODES_TO_PROPAGATE_IN_SIMORGH,
 } from './utils/statusCodes';
+import sendCustomMetric from '#lib/utilities/customMetrics';
```

Comment:
This will now be included in the client side bundle will it not?

We might want to try and restrict that and ensure the sendCustomMetric function is only called on the server

Reply:
Shall I do an `onClient()` check + dynamic import?

Reply:
would it still get included even with the dynamic import? not 100% how they work tbh :(

Reply:
we'd have to do the `onClient()` check either way

Reply:
Hopefully this is now resolved (experimented with dynamic import but tests failed, so resorted to `require` instead)